quickDeploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+INSTALL=false
+MONGOPATH="data/db"
+FIFACHAMPS="$(echo $PWD)"
+STOP=false
+
+for i in "$@"
+do
+  case ${i} in
+    -i|--install)
+      INSTALL=true
+      shift ;;
```

Comment:
is this shift really needed here? :thinking: 

I know the one at line 19 is, but I think all the others are not

Reply:
makes sense. removed unnecessary shifts.